src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOMIZER_BOUND = 100;
+    private final SecureRandom secureRandom = new SecureRandom();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (FigureName.values()[secureRandom.nextInt(FigureName.values().length)]) {

```

Comment:
please introduce a local variable for `FigureName.values()[secureRandom.nextInt(FigureName.values().length)]`, that will improve readability.